### ISSUE TITLE
email.contentmanager.raw_data_manager fails to create multipart messages

### ISSUE THREAD

**Issue by cdb25775-6a9d-484e-b13e-4e0fc806c5ad** [python/cpython#75003]
BPO | [30820](https://bugs.python.org/issue30820)
--- | :---
Nosy | @warsaw, @bitdancer, @elenril, @ZackerySpytz
PRs | <li>python/cpython#7631</li><li>python/cpython#7633</li><li>python/cpython#7634</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = <Date 2018-06-11.16:12:23.991>
created_at = <Date 2017-06-30.17:03:41.070>
labels = ['3.8', 'type-bug', '3.7', 'expert-email']
title = 'email.contentmanager.raw_data_manager fails to create multipart messages'
updated_at = <Date 2018-06-11.16:12:23.990>
user = 'https://github.com/elenril'
```

bugs.python.org fields:
```python
activity = <Date 2018-06-11.16:12:23.990>
actor = 'r.david.murray'
assignee = 'none'
closed = True
closed_date = <Date 2018-06-11.16:12:23.991>
closer = 'r.david.murray'
components = ['email']
creation = <Date 2017-06-30.17:03:41.070>
creator = 'elenril'
dependencies = []
files = []
hgrepos = []
issue_num = 30820
keywords = ['patch']
message_count = 6.0
messages = ['297430', '297439', '319307', '319313', '319314', '319315']
nosy_count = 4.0
nosy_names = ['barry', 'r.david.murray', 'elenril', 'ZackerySpytz']
pr_nums = ['7631', '7633', '7634']
priority = 'normal'
resolution = 'fixed'
stage = 'resolved'
status = 'closed'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue30820'
versions = ['Python 3.6', 'Python 3.7', 'Python 3.8']
```

</p></details>

**Comment by cdb25775-6a9d-484e-b13e-4e0fc806c5ad** [comment]
The documentation for the "new API" -- email.contentmanager.raw_data_manager -- claims that passing a list of messages to set_content() will create a multipart message. However, it fails with "KeyError: 'builtins.list'" and from looking at the code it seems this functionality is not and never was implemented.

**Comment by bitdancer** [comment]
We should fix the docs for 3.5 and 3.6, and open an enhancement request for 3.7 if we still think this feature is a good idea.
